tools/nnyeah/integration/Makefile

```diff
@@ -0,0 +1,27 @@
+TOP=../../..
+include $(TOP)/Make.config
+
+all-local:: mac/bin/Debug/MacIntegration.dll ios/bin/Debug/iOSIntegration.dll tv/bin/Debug/TVIntegration.dll
+
+install-local:: all-local
+
+run-tests::
+	$(Q) rm -rf converted
+	$(Q) mkdir -p converted	
+	-$(Q) $(SYSTEM_DOTNET) run --project ../nnyeah/nnyeah.csproj --input=mac/bin/Debug/MacIntegration.dll --output=Converted/MacIntegration.dll
+	-$(Q) $(SYSTEM_DOTNET) run --project ../nnyeah/nnyeah.csproj --input=ios/bin/Debug/iOSIntegration.dll --output=Converted/iOSIntegration.dll
+	-$(Q) $(SYSTEM_DOTNET) run --project ../nnyeah/nnyeah.csproj --input=tv/bin/Debug/TVIntegration.dll --output=Converted/TVIntegration.dll	
```

Comment:
By prepending the dash, you're preventing detection of failures from scripts ("make run-tests" will always succeed, even if there are test failures).

Reply:
Yep, I did that because they are broken (see https://github.com/xamarin/xamarin-macios/pull/14902).